src/main/scala/io/iohk/ethereum/network/PeerActor.scala

```diff
@@ -177,19 +180,19 @@ class PeerActor(
       daoBlockHeaderOpt match {
         case Some(_) if daoForkValidator.validate(msg).isEmpty =>
           log.info("Peer is running the ETC chain")
-          context become new HandshakedHandler(rlpxConnection, remoteStatus).receive
+          context become new HandshakedHandler(rlpxConnection, remoteStatus, daoForkBlockNumber).receive
 
         case Some(_) if nodeStatusHolder().blockchainStatus.totalDifficulty < daoForkBlockTotalDifficulty =>
           log.info("Peer is not running the ETC fork, but we're not there yet. Keeping the connection until then.")
-          context become new HandshakedHandler(rlpxConnection, remoteStatus).receive
+          context become new HandshakedHandler(rlpxConnection, remoteStatus, daoForkBlockNumber).receive
```

Comment:
@LukasGasior1 do we have mechanism that prevent us from starting fast sync from this peer?